### PR TITLE
added gsoc message

### LINKED ISSUE #423: Add FAQ entry to say coala will apply to GSOC 2018
And add a link in FAQ to https://developers.google.com/open-source/gsoc/timeline
Opened by @jayvdb at [Gitter](https://gitter.im/coala/coala/gsoc?at=5a0b062ecc1d527f6b6c6ca7)

### PR DESCRIPTION
**Pull request by ihsraham**
I have added gsoc message in FAQ

Closes #423 

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by yashovardhanagrawal** [comment]
first, read our commit guidelines
you shortlog is wrong and also you have not added a commit description the main reason why the last PR for this issue was closed

File: _faq/coala_will_apply_for_gsoc_2018.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+question: "coala will apply to GSOC 2018"
+---
```
Comment (yashovardhanagrawal) [inline _faq/coala_will_apply_for_gsoc_2018.md]: this is totally wrong

Comment (yashovardhanagrawal) [reply _faq/coala_will_apply_for_gsoc_2018.md]: for the question part write
`Will coala apply for GSoC 2018?`

Comment (yashovardhanagrawal) [reply _faq/coala_will_apply_for_gsoc_2018.md]: for its answer you have to mention the timeline

Comment (yashovardhanagrawal) [reply _faq/coala_will_apply_for_gsoc_2018.md]: and a positive message
like
> Yes, coala will apply for Google Summer of Code - 2018. You might read the (GSoC Timeline)[https://developers.google.com/open-source/gsoc/timeline] to get an idea of how things will proceed.

**Comment by yashovardhanagrawal** [comment]
A better shortlog can be: 
`coala_will_apply_for_gsoc_2018.md: Add new FAQ`

Description:
> This adds a new FAQ related to coala applying for
> Google Summer of Code - 2018
>
> Closes <issue link>

**Comment by yashovardhanagrawal** [comment]
Also forgot to mention, rebase your commit to update the changes

you might read http://api.coala.io/en/latest/Developers/Git_Basics.html for more information about how to do it

File: _faq/coala_will_apply_for_gsoc_2018.md (hunk 1 above)
Comment (ihsraham) [reply _faq/coala_will_apply_for_gsoc_2018.md]: Oh sorry I forgot to add the timeline there

will do it as you said

**Comment by ihsraham** [comment]
thanks a lot @yashovardhanagrawal I know how to rebase a commit, going to do it now 👍

**Comment by ihsraham** [comment]
@yashovardhanagrawal done with the suggested changes

**Comment by ihsraham** [comment]
@prashant0598 will keep that in mind from the next time, in this PR I cannot change the branch and I think closing a PR would not be good. 
@yashovardhanagrawal I have again updated a few changes

You both may review it

**Comment by prashant0598** [comment]
ack f80b9ac

**Comment by yashovardhanagrawal** [comment]
@ihsraham do not close valid PRs

**Comment by yashovardhanagrawal** [comment]
@jayvdb he might have closed it by mistake... you people reopen it

**Comment by prashant0598** [comment]
@ihsraham pls remember never to close PR  on your own

**Comment by jayvdb** [comment]
@ihsraham also deleted his fork, so it isnt possible to re-open this.

**Comment by jayvdb** [comment]
@rultor merge

**Comment by ihsraham** [comment]
sorry, by mistake I deleted my fork thats why this happened 😟

**Pull request closed without merging**